cdap-docs/faq/source/cdap-startup.rst

```diff
@@ -0,0 +1,226 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+:titles-only-toc: true
+
+.. _faq-installation-startup:
+
+======================
+CDAP FAQ: CDAP Startup
+======================
+
+.. contents::
+   :depth: 2
+   :local:
+   :backlinks: entry
+   :class: faq
+
+
+Startup
+=======
+
+My CDAP install on CDH using Cloudera Manager doesn't startup |---| what do I do?
+---------------------------------------------------------------------------------
+We have a :ref:`tutorial <step-by-step-cloudera-add-service>` with instructions on how to install CDAP on CDH 
+(`Cloudera Data Hub <http://www.cloudera.com/content/www/en-us/resources/datasheet/cdh-datasheet.html>`__) 
+using `Cloudera Manager <http://www.cloudera.com/content/www/en-us/products/cloudera-manager.html>`__. 
+
+If, when you try to start services, you receive an error in ``stderr`` such as::
+       
+  Error found before invoking supervisord: No parcel provided required tags: set([u'cdap'])
+
+The error message suggests that you have not completed the last step of installing a
+parcel, *Activation*. There are 4 steps to installing a parcel:
+
+- **Adding the repository** to the list of repositories searched by Cloudera Manager
+- **Downloading** the parcel to the Cloudera Manager server
+- **Distributing** the parcel to all the servers in the cluster
+- **Activating** the parcel
+
+Start by clicking on the parcel icon (near the top-left corner of Cloudera Manager; looks
+like a gift-wrapped box), and ensure that the CDAP parcel is listed as *Active*.
+
+
+I've followed the install instructions, yet CDAP does not start and fails verification. What next?
+--------------------------------------------------------------------------------------------------
+If you have followed :ref:`the installation instructions <install>`, and CDAP either did not pass the 
+:ref:`verification step <configuration-verification>` or did not startup, check:
+
+- Look in the CDAP logs for error messages (located in ``$CDAP_HOME/logs``)
+- If you see an error such as::
+
+    ERROR [main:c.c.c.StandaloneMain@268] - Failed to start Standalone CDAP
+    java.lang.NoSuchMethodError: 
+    co.cask.cdap.UserInterfaceService.getServiceName()Ljava/lang/String
+
+  then you have probably downloaded the standalone version of CDAP, which is not intended
+  to be run on Hadoop clusters. Download the appropriate distributed packages (RPM or
+  Debian version) from http://cask.co/downloads.
+         
+- Check permissions of directories:
+
+  - The :ref:`CDAP HDFS User <configuration-options>` (by default, ``yarn``) owns the HDFS directory (by default,  ``/cdap``).
+  - The :ref:`Kafka Log directory <configuration-options>` (by default, ``/data/cdap/kafka-logs``), must be writable by the default CDAP user.
+  - The :ref:`temp directories <configuration-tmp-files>` (by default, ``/tmp`` and ``/tmp/kafka-logs``) utilized by CDAP must be writable by the default CDAP user.
+
```

Comment:
Added that in another PR, and have changed this to not include specific directories, as that may be a moving target.